includes/Core/Modules/Modules.php

```diff
@@ -1119,4 +1127,45 @@ function( Module $module ) {
 		);
 	}
 
+	/**
+	 * Gets the recoverable modules.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array List of recoverable modules.
+	 */
+	public function get_recoverable_modules() {
+		$recoverable_modules = array();
+		$shareable_modules   = $this->get_shareable_modules();
+
+		foreach ( $shareable_modules as $module ) {
+			$owner_id = $module instanceof Module_With_Owner && $module->get_owner_id();
+
+			// If no owner identified by its owner_id OR lacks the AUTHENTICATE capability -
+			// push the module slug to the recoverableModules array.
+			if ( false === $owner_id || ! user_can( $owner_id, Permissions::AUTHENTICATE ) ) {
+				$recoverable_modules[] = $module->slug;
+				continue;
+			}
+
+			// If the owner doesn't have the AUTHENTICATE permission - push the module slug to the recoverableModules array.
+			$user = get_user_by( 'ID', $owner_id );
+
+			// If the User doesn't exists - push the module slug to the recoverableModules array.
+			if ( false === $user ) {
+				$recoverable_modules[] = $module->slug;
+				continue;
+			}
```

Comment:
I think we don't need this check anymore because the `user_can` function verifies that the user exists. If the user doesn't exist anymore, the `user_can` will return `false` and the previous condition will be satisfied. So we can remove this section.